application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -92,12 +96,20 @@ public void onMessageContext(MessageContextInteractionEvent event) {
         String originalChannelId = event.getTarget().getChannel().getId();
         String authorId = event.getTarget().getAuthor().getId();
         String mostCommonTag = tags.get(0);
+        String chatGptPrompt = String.format(
+                "Generate topic for question: %s. Minimum %s and maximum %s characters.",
+                originalMessage, TITLE_MIN_LENGTH, TITLE_MAX_LENGTH);
```

Comment:
```suggestion
        String chatGptPrompt =
                "generate title for this question %s, it shouldn't be longer than 4-5 words with no header".formatted(originalMessage);
```

I suggest using `formatted` instead, it's same thing but for consistency across project it's a bit better.

Also made slight variations in prompt, gpt isn't too great with character limits so there was no point in using Max or min limit here. When i tried running the prompt you added, it would add `Title: whatever title here` as header sometimes also the title was for some reasons inside double quotes.

i tested this prompt with couple of questions and generates more cleaner ones, you can try it out as well.

Reply:
for reference this is how earlier title was generated with older prompt
![image](https://github.com/Together-Java/TJ-Bot/assets/61616007/64435bd1-9462-4580-92e6-89cedb17d380)